### DISCUSSION TITLE
Angry at Github Education's automated review system, submissions are rejected within seconds

### DISCUSSION THREAD

**Discussion by Cetaceaqua** [community/community#58014] [GitHub Education]
### Hi everyone,

I am a university student from China.

I have successfully applied for the Github Education student pack in 2020 and 2021 by submitting photos of my student ID and other documents.

However, since Github Education has updated to the current application system, I have been unable to renew my Github Education student pack.

I submitted my student ID, student card, acceptance letter, and CHSI proof of enrollment from the government website via webcam shot, and then even tried to attach English translations to these images using the OBS virtual webcam, but all dozens of applications were never reviewed manually and were rejected by the automated system in mere seconds.

I was able to apply for student discounts from companies like Apple, JetBrains, AutoDesk, and Unity using the same, real school documents, but was completely unable to submit them to the staff at Github for review.

Github also does not review Github Education related Github Support Tickets at all, which is very disappointing.

I would really like to be able to continue using the Github Copilot subscription included with the student package to improve the quality of my code, and I hope that Github will soon improve or remove the current automated review system for Github Education so that real students can apply for the student package benefits.

**Comment by samus-aran** [comment]
Hi @Cetaceaqua !

First off I want to thank you for sharing your experiences and that we absolutely understand your frustration. We are trying our best to continually improve our system to be as easy as possible for legitimate students everywhere to verify their academic status, while also ensuring that bad actors impersonating students don't jeopardise our ability to provide academic benefits of value.

As you can imagine there are many types of academic verification and our system is constantly evolving to accommodate this without setting the bar too high. However sometimes we may require a little more evidence from the student applying and we thank you for being patient.

Ensure you select the most appropriate **evidence type** in the dropdown. The order of options indicates the preference of evidence with Academic Email being at the top. If you don't have an Academic email or Student ID, do try and acquire evidence that is positioned higher rather than lower in that option list.

If you have been repetitively unsuccessful using a particular evidence type (e.g. Student ID) we would recommend that you acquire and submit an alternative piece of evidence that can verify your student status.

Hope this helps 👍 Let us know how you get on!
